_inc/lib/class.media-summary.php

```diff
@@ -63,11 +63,34 @@ static function get( $post_id, $blog_id = 0, $args = array() ) {
 				switch ( $type ) {
 					case 'wpvideo':
```

Comment:
We should add in a fall-through for `case 'videopress':` as well, so that it shares this logic.